src/sql/base/browser/ui/radioButton/radioButton.ts

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+import * as DOM from 'vs/base/browser/dom';
+import { IContextViewProvider } from 'vs/base/browser/ui/contextview/contextview';
+import { Color } from 'vs/base/common/color';
+import Event, { Emitter } from 'vs/base/common/event';
+
+export class RadioButton {
```

Comment:
I don't think this is necessary. We can reuse `InputBox` and just set its input type to `radio`.

Reply:
What didn't it have? Behind the scenes the HTML should be the same so I'm surprised it wouldn't. Maybe this class could extend it instead of reimplementing things?

Reply:
if you're talking about the vs/base/browser/ui/inputbox/inputBox that class didn't have the checked proeprty or other thing s I needed for radio buttonm I could add the property there but textbox won't need that so I created a separate class for radio button with it's own propertied 

---
In reply to: [189119253](https://github.com/Microsoft/sqlopsstudio/pull/1439#discussion_r189119253) [](ancestors = 189119253)